designs/extensible-cli-and-scaffolding-plugins-phase-1.md

```diff
@@ -170,8 +170,14 @@ Example Operator SDK main.go:
 
 ```go
 func main() {
+    rootCmdCfg := cli.RootCommandConfig{
+        CommandName: "operator-sdk",
+        //Short: "Short help text for operator-sdk",
```

Comment:
I think we should not use SDK as a comment here. So, wy about update the comments with something such as `//Short: "Short help text for an external tool which is a consumer of kubebuilder",`?